pkgs/by-name/ra/ranger/package.nix

```
@@ -13,18 +13,28 @@
   improvedEncodingDetection ? true,
   rightToLeftTextSupport ? false,
   gitUpdater,
+  rangerGit ? false,
+  videoThumbnailSupport ? false, # you have to have rangerGit enabled
+  ffmpegthumbnailer,
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "ranger";
-  version = "1.9.4";
+  version = if rangerGit then "master" else "1.9.4"; # master as of 2025/05/25
```

```suggestion
  version = if rangerGit then "1.9.4-unstable-2025-05-25" else "1.9.4"; # master as of 2025/05/25
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

Furthermore, do you think this change is really suitable for inclusion in Nixpkgs?